service/volume.py

```diff
@@ -8,10 +6,13 @@
 from core.models.identity import Identity
 
 from service.cache import get_cached_driver
-from service.driver import _retrieve_source
+from service.driver import _retrieve_source, prepare_driver
 
+from service import exceptions
 from service.instance import boot_volume_instance
-from service.exceptions import OverQuotaError
+
+# FIXME: fix prepare_driver to take a user directly
+Request = namedtuple("request", ["user"])
```

Comment:
'prepare_driver' was actually a derivative of https://github.com/iPlantCollaborativeOpenSource/atmosphere/blob/volume_v2/service/driver.py#L122-L138 -- Perhaps we should modify that function to match the behavior you are looking for, and do away with 'prepare_driver' altogether, since the WSGIRequest is inconsequential here